charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -43,20 +43,21 @@ spec:
         - name: scalardl-ledger-properties-volume
           configMap:
             name: {{ include "scalardl.fullname" . }}-ledger-properties
-      {{- if .Values.ledger.tls.caRootCertSecret }}
-        - name: scalardl-ledger-tls-ca-root-volume
+      {{- if and .Values.ledger.tls.enabled .Values.ledger.tls.certManager.enabled }}
+        - name: scalardl-ledger-tls-volume
           secret:
-            secretName: {{ .Values.ledger.tls.caRootCertSecret }}
+            secretName: {{ include "scalardl.fullname" . }}-tls-cert
       {{- end }}
-      {{- if .Values.ledger.tls.certChainSecret }}
-        - name: scalardl-ledger-tls-cert-chain-volume
-          secret:
-            secretName: {{ .Values.ledger.tls.certChainSecret }}
-      {{- end }}
-      {{- if .Values.ledger.tls.privateKeySecret }}
-        - name: scalardl-ledger-tls-private-key-volume
-          secret:
-            secretName: {{ .Values.ledger.tls.privateKeySecret }}
+      {{- if and (.Values.ledger.tls.enabled) (not .Values.ledger.tls.certManager.enabled) }}
+        - name: scalardl-ledger-tls-volume
+          projected:
+            sources:
+              - secret:
+                  name: {{ .Values.ledger.tls.caRootCertSecret }}
+              - secret:
+                  name: {{ .Values.ledger.tls.certChainSecret }}
+              - secret:
+                  name: {{ .Values.ledger.tls.privateKeySecret }}
       {{- end }}
```

Comment:
If you don't use cert-manager (and you want to manage each file separately), you have to create three secrets for the private key, certificate, and CA certificate.

In this case, I want to combine them into one volume to mount them easily under the same directory in the container. So, I use `projected` volume here.